docs/app/Examples/modules/Progress/Content/ProgressExampleLabel.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { Progress } from 'semantic-ui-react'
 
 const ProgressExampleLabel = () => (
-  <Progress percent={55}>Label</Progress>
+  <Progress percent={55} label>Label</Progress>
```

Comment:
The label prop is not required here.  This is confusing on SUI core's part, however, the label prop controls the label **inside of the bar** while the `Label` child here is the label **below the progress bar**:

![image](https://cloud.githubusercontent.com/assets/5067638/23145472/4efbdd24-f783-11e6-823c-4601dfb2ff8d.png)


I feel like these should almost have separate names so it is more clear that there are two different label positions (in the bar, under the bar).